classes/log/EventLogEntry.inc.php

```diff
@@ -166,6 +168,46 @@ function getTranslatedMessage($locale = null) {
 
 		$params = array_merge($this->_data, $this->getParams());
 		unset($params['params']); // Clean up for translate call
+
+		if ($hideReviewerName) {
```

Comment:
Points for thoroughness!

Reply:
Personally, I might've just denied access to the log in situations where it might violate blindness -- but I have no objection to this. We'll just need to ensure that we keep in mind when changing/adding log entries.